application/resources/views/tasks/edit.blade.php

```diff
@@ -1,29 +1,65 @@
 @section('script')
 <script>
-    function toggleModal() {
+    function toggleModal(event) {
         const body = document.querySelector('body');
-        const modal = document.querySelector('.modal');
+        const modal = document.querySelectorAll('.modal');
+        const selectModal = event.currentTarget.getAttribute('data-target');
```

Comment:
```suggestion
        const selectedModal = event.currentTarget.getAttribute('data-target');    
```

細かい命名ですが、selectは動詞なので、〜されている / 〜のを説明する場合は過去分詞を使うことが一般的です

Reply:
A.selectedに修正しました。